packages/design-system/src/styles/components/_Alert.scss

```diff
@@ -6,7 +6,7 @@
   box-sizing: border-box;
   color: $alert__font-color;
   display: flex;
-  min-height: $alert__icon-size + $spacer-2;
```

Comment:
I actually don't think we even need this `min-height` property. It seems to be for the case where no content is in the alert, which is not a valid use of our component. The other thing is that when I remove it, it seems to stand up just fine without conent, probably because of structural changes that have happened since this was first written.

<img width="208" alt="Screen Shot 2022-12-06 at 11 22 20 AM" src="https://user-images.githubusercontent.com/7595652/206003355-b6582a75-dde7-4d35-9c3a-62ff896527d0.png">